src/transport_picoquic.cpp

```diff
@@ -1161,7 +1173,7 @@ void PicoQuicTransport::on_recv_stream_bytes(StreamContext* stream_cnx, uint8_t*
     uint8_t *bytes_p = bytes;
 
     if (stream_cnx == NULL || length == 0) {
-        logger->Log(cantina::LogLevel::Warning, "on_recv_stream_bytes has null context");
+        logger->Log(cantina::LogLevel::Debug, "on_recv_stream_bytes has null context");
```

Comment:
Since you want this to be debug, perhaps use the `LOGGER_DEBUG` macro so this line can be compiled out? It won't matter much if this line is rarely hit, but if seen often it will make a difference in production code.

Reply:
Thanks. I think there are a few places we need to update the logging. Might be best to do that as a logging PR.